server/src/main/java/org/opensearch/discovery/InitializeExtensionResponse.java

```diff
@@ -68,21 +75,29 @@ public String getName() {
         return this.name;
     }
 
+    /**
+     * @return interfaces implemented by an extension
+     */
+
+    public List<String> getImplementedInterfaces() {
+        return implementatedInterfaces;
+    }
+
     @Override
     public String toString() {
-        return "PluginResponse{" + "name" + name + "}";
+        return "PluginResponse{" + "name" + name + "implementedInterfaces" + implementatedInterfaces + "}";
```

Comment:
need a comma and space after name, before the next string.  Also probably want an `=` after both name and interfaces.